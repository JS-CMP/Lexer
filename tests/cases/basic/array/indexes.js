var array = [1, 2, 3];
var b = [1];

array[1] = 4;
array[b[0]] = 5;
((array))[2] = 6;