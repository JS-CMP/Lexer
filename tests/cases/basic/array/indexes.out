#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any array, b;array=JS::Any(JS::Array({JS::Any(1), JS::Any(2), JS::Any(3)}));b=JS::Any(JS::Array({JS::Any(1)}));array[JS::Any(1)]=JS::Any(4);array[b[JS::Any(0)]]=JS::Any(5);((array))[JS::Any(2)]=JS::Any(6);}
